demo/assets/materials/depth.material

```diff
@@ -12,7 +12,11 @@
                     shader_module: "../shaders/depth.vert.cookedshaderpackage",
                     entry_name: "main"
                 ),
-                // No fragment shader required
+                (
```

Comment:
I want to make the GL ES 3.0 use a default shader automatically in this case so an "empty" frag shader doesn't need to be provided